_posts/2019-11-08-method-references-the-unpolished-gem-of-java-8.markdown

```diff
@@ -295,5 +295,5 @@ I recommend using Method References whenever possible.
 Your code will become cleaner and easier to refactor, at no extra cost.
 Keep in mind though that they have some limitations and unless those get waived in the next Java releases, some workarounds need to be implemented.
 
-Method references are—in my humble opinion—not a complete feature.
-There are some improvements that could be added to allow wider use of them.
\ No newline at end of file
+Method references are —in my humble opinion— not a complete feature.
```

Comment:
Without the spaces it was hard for me to read, but it doesn't mean it's correct with the spaces, in this regard (and the its') I'd ask Huw

Reply:
There's not standard for this I think, it's a style preference - https://english.stackexchange.com/questions/214585/space-before-after-emdash-or-not (:

These dashes will look differently if you view them in a rendered markdown. The size is off with mono font.